portfolio/src/main/webapp/index.html

```diff
@@ -64,6 +57,11 @@ <h2 class="hello-text">Hello,</h2>
           my GitHub.
         </p>
       </div>
+
+      <div id="step2">
```

Comment:
style convention is hyphenated spacing

Reply:
can you please elaborate?

Reply:
step2 is not a single word but 2! step-2? step-two? 🤔

Reply:
oh okay, fixed